packages/manager/cypress/e2e/core/cloudpulse/quotas.spec.ts

```diff
@@ -0,0 +1,87 @@
+import { mockAppendFeatureFlags } from 'support/intercepts/feature-flags';
+import { ui } from 'support/ui';
+
+describe('Quotas accessible when limitsEvolution feature flag enabled', () => {
+  beforeEach(() => {
+    // TODO M3-10003 - Remove mock once `limitsEvolution` feature flag is removed.
+    mockAppendFeatureFlags({
+      limitsEvolution: {
+        enabled: true,
+      },
+    }).as('getFeatureFlags');
+  });
+  it('can navigate directly to Quotas page', () => {
+    cy.visitWithLogin('/account/quotas');
+    cy.wait('@getFeatureFlags');
+    cy.url().should('endWith', '/quotas');
+    cy.findByText('Object Storage').should('be.visible');
```

Comment:
```suggestion
    cy.contains('View your Object Storage quotas by applying the endpoint filter below').should('be.visible');
```

Since "Object Storage" exists in the side bar nav, this test passes even if I disable the Limits Evolution mock on line 9. Changing this assertion to something that only exists on the quotas page gives us the behavior we want (it doesn't have to be the specific blurb I used above)

Reply:
> would it be a good practice to wrap some or all my test content in cy.get('#main-content).within(() => { });

If that narrows the selection down enough to reliably select what you need then I think it's fine to do 👍 we often set an ARIA label on our tables which is nice for screen reader users and allows us to use `cy.findByLabelText("List of [...]")` to select them in our tests, but I notice this table doesn't have one. You could add one if you want to, or do `cy.get('#main-content').within(...)`, or just select some text that's unique to the quotas tab -- either would solve the problem and I think they're all fine solutions.